DOCS-13079 Update API (#62)
mulesoft/docs-anypoint-datagraph: modules/ROOT/pages/promote-api.adoc

* *Custom*: Use if you authenticate to your API using custom header parameters and values.
. Click *Next: Add to unified schema*.

== Promote an Updated API 

> After you've update an API schema to DataGraph, you can later update the version of the API source. 

Author: good catch. I'll fix it

> After you've update an API schema to DataGraph, you can later update the version of the API source. 

Author: ```suggestion
After you've updated an API version, you can promote the updated version to another environment. 
```

When you choose a new version of an API to promote, DataGraph lists any edits you've made to the current version that it can't automatically apply to the new version in the target environment.

> For any edits DataGraph can't automatically apply, you'll have the option to manually apply these edits before you complete the promotion. 

Reviewer: Ditto prev. alternative

> Before you promote an updated API schema, you must first xref:add-api-to-unified-schema.adoc#upate-an-api-version[update the API version].

Reviewer: ```suggestion
Before you promote an updated API schema, you must first xref:add-api-to-unified-schema.adoc#upate-an-api-version[update the API version]:
```

. Click *List of APIs added* and select the appropriate API.
. Click *API details*.
. Click *Promote API schema*.
. From the *Target environment* list, select the target environment for promotion.
. Click *Next*.
. If DataGraph finds changes it cannot apply, review the list of *Changes Not Applied* and manually apply them: 
.. Select a change and click *Next*.
.. Edit the API schema as necessary and click **Apply Changes and Promote**. 
+

> After you apply the change, repeat step 5 for any additional changes.

Reviewer: ```suggestion
After you apply the change, repeat Step 5 for any additional changes.
```
